.github/workflows/scripts/builder-fetch.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Caller sets the following:
+#BUILDER_REPOSITORY="slsa-framework/slsa-github-generator"
+#BUILDER_TAG="v13.0.10" or "6a1e642a8689671a2cec9287149eb50bd9fe5ef6"
+#BUILDER_RELEASE_BINARY="builder-linux-amd64"
+#VERIFIER_REPOSITORY="slsa-framework/slsa-verifier"
+#VERIFIER_RELEASE="v13.0.10"
+#VERIFIER_RELEASE_BINARY="slsa-verifier-linux-amd64"
+#VERIFIER_RELEASE_BINARY_SHA256="89fbcba9aed67d5146ea99946c7e4e5a80e3767871f0e3ffcd0b582134efd010"
+
+PREFIX="refs/tags/"
+
+# Extract version.
+if [[ "$BUILDER_REF" =~ "^$PREFIX*" ]]; then
+    echo "Invalid ref: $BUILDER_REF"
+    exit 2
+fi
+
+BUILDER_TAG="${BUILDER_REF#"$PREFIX"}"
+
+if [[ "$BUILDER_TAG" = "$(echo -n "$BUILDER_TAG" | grep -P '^[a-f\d]{40}$')" ]]; then
+    echo "Builder referenced by hash: $BUILDER_TAG"
+    echo "Resolving..."
+    
+    RELEASE_TAG=""
+
+    # List the releases and find the corepsonding hash.
+    RELEASE_LIST=$(gh release -R "$BUILDER_REPOSITORY" -L 50 list)
```

Comment:
is `gh` always in GH workflow runners?

Reply:
yes it is. it's a temp solution though, we'll replace the code with Go code https://github.com/slsa-framework/slsa-github-generator/issues/87